cms/djangoapps/contentstore/views/program.py

```diff
@@ -38,3 +39,16 @@ def get(self, request, *args, **kwargs):
             })
         else:
             raise Http404
+
+
+class ProgramsIdTokenView(View):
+    """Provides id tokens to JavaScript clients for use with the Programs API."""
+
+    @method_decorator(login_required)
```

Comment:
Nope, I guess not. To keep things consistent, it might be worth updating ProgramAuthoringView as well (which also only implements GET).